internal/cluster/connection_manager_test.go

```diff
@@ -0,0 +1,126 @@
+package cluster
+
+import (
+	"context"
+	"errors"
+	pubcluster "github.com/hazelcast/hazelcast-go-client/cluster"
+	"github.com/hazelcast/hazelcast-go-client/internal"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type CheckedAddressHelper struct {
+	error   error
+	address pubcluster.Address
+}
+
+func TestTryConnectAddress_NoAddressAvailable(t *testing.T) {
+	// inputs
```

Comment:
Can you remove comments like: inputs, perform, assert? They don't seem to be necessary.